manifests/init.pp

```diff
@@ -18,6 +18,12 @@
 # $host_private_key:: path to host private key, usually puppet agent private key
 #                     May be overriden if $::rh_certificate_consumer_host_key (from Facter) is found
 #
+# $foreman_repo_rel:: add / manage foreman-plugins yum repo and set to release version. Eg  '1.14'
+#                     type: string
+#
+# $foreman_rpm_key::  RPM Key source file for foreman-plugins repo. Note: Currently, packages are not signed
+#                     type: string
```

Comment:
could you please add a note that this can be url such as "https://yum.theforeman.org/RPM-GPG-KEY-foreman"